appveyor.yml

```diff
@@ -0,0 +1,19 @@
+version: '{build}'
+image: Visual Studio 2019
+build_script:
+- pwsh: >-
+    mkdir build
```

Comment:
are these additional newlines here an artifact of `\r\n` on windows?

Reply:
If we remove those AppVeyor will not build at all. Yes, I have tried it.